packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -1,68 +1,73 @@
-                :class="$style['c-checkout-form']"
-                @submit.prevent="onFormSubmit">
-                <guest-block v-if="!isLoggedIn" />
-
-                <form-field
-                    :value="customer.mobileNumber"
-                    name="mobile-number"
-                    :label-text="$t('labels.mobileNumber')"
-                    :has-error="!isMobileNumberValid"
-                    @input="updateCustomerDetails({ mobileNumber: $event })">
-                    <template #error>
-                        <error-message
-                            v-if="!isMobileNumberValid"
-                            data-test-id="error-mobile-number">
-                            {{ $t('validationMessages.mobileNumber.requiredError') }}
-                        </error-message>
-                    </template>
-                </form-field>
-
-                <address-block
-                    v-if="isCheckoutMethodDelivery"
-                    data-test-id="address-block" />
-
-                <form-selector />
-
-                <user-note data-test-id="user-note" />
-
-                <f-button
-                    :class="$style['c-checkout-submitButton']"
-                    button-type="primary"
-                    button-size="large"
-                    is-full-width
-                    action-type="submit"
-                    data-test-id="confirm-payment-submit-button">
-                    {{ $t('buttonText') }}
-                </f-button>
-            </form>
-
-            <checkout-terms-and-conditions
-                v-if="!isLoggedIn" />
-        </card>
+    <div>
+        <div
+            v-if="hasCheckoutLoadedSuccessfully"
+            data-theme="jet"
+            data-test-id="checkout-component">
+            <alert
+                v-if="genericErrorMessage"
+                type="danger"
+                :class="$style['c-checkout-alert']"
+                :heading="$t('errorMessages.errorHeading')">
+                {{ genericErrorMessage }}
+            </alert>
+
+            <card
+                is-rounded
+                has-outline
+                is-page-content-wrapper
+                card-heading-position="center"
+                data-test-id="checkout-card-component"
+                :class="$style['c-checkout']">
+                <checkout-header
+                    :login-url="loginUrl" />
+
+                <form
+                    method="post"
+                    :class="$style['c-checkout-form']"
+                    @submit.prevent="onFormSubmit">
+                    <guest-block v-if="!isLoggedIn" />
+
+                    <form-field
+                        :value="customer.mobileNumber"
+                        name="mobile-number"
+                        :label-text="$t('labels.mobileNumber')"
+                        :has-error="!isMobileNumberValid"
+                        @input="updateCustomerDetails({ ['mobileNumber']: $event })">
```

Comment:
I think the square brackets and quotes can be removed again

Reply:
Ah, it was probably a conflict, yes. Will take a look, thanks.